composer.json

```diff
@@ -4,7 +4,7 @@
   "require": {
     "symfony/yaml": "2.*",
     "symfony/console": "2.*",
-    "webmozart/path-util": "~1.1"
+    "webmozart/path-util": "^2.3"
```

Comment:
This should be `~2.3` to stay in semver and not accidently upgrade to `3.*` or later.